components/annot-vocabulary.vue

```diff
@@ -73,14 +68,8 @@
 
         props: {
 
-            filteredDataTable: { type: Array, default: () => [] },
-
-            // Options contains 'mode' which is one of ["column", "row"]
-            // column: Only the column names are mapped to vocabulary terms inside the data dictionary
-            // row: The row values are mapped to vocabulary terms
-            options: { type: Object, required: true },
-            relevantColumns: { type: Array, required: true },
-            uniqueValues: { type: Object, required: true }
+            relevantColumns: { default: () => [], required: true, type: Array },
+            uniqueValues: { default: () => {}, required: true, type: Object }
```

Comment:
nice! cleanup!
But same as above, I don't think we need to / should set defaults for required props. We'd like vue to yell at folks for us if they don't provide required props, at a minimum, setting defaults for this could be confusing, worst case it shuts up vue-yelling in cases where folks forget to provide the prop.